plugins/star-rating/api/api.js

```diff
@@ -119,6 +437,7 @@ var plugin = {},
         return false;
     });
 
+
```

Comment:
Also I don't see handling of app lifecycle, adding indexes if needed on app create, deleting data on app clear/reset/delete

Giving examples here:
App create: https://github.com/Countly/countly-server/blob/master/plugins/crashes/api/api.js#L1173-L1182
App delete: https://github.com/Countly/countly-server/blob/master/plugins/crashes/api/api.js#L1184-L1193
App clear specific period: https://github.com/Countly/countly-server/blob/master/plugins/crashes/api/api.js#L1195-L1202
App clear all data: https://github.com/Countly/countly-server/blob/master/plugins/crashes/api/api.js#L1204-L1222
App reset: https://github.com/Countly/countly-server/blob/master/plugins/crashes/api/api.js#L1224-L1242